ui/src/theme/globals.ts

```diff
@@ -1,18 +1,29 @@
 import { globalCss } from '@stitches/react';
 
 export const themeGlobals = globalCss({
-  'html, body, #root': {
+  'html, body': {
     height: '100%',
-    padding: 0,
+
+    //TODO add theme colors
     color: '#ECEDEE',
     backgroundColor: 'black',
 
     fontFamily: 'Manrope',
-
     fontSize: '16px',
 
+    '* > *': {
+      boxSizing: 'border-box',
+    },
+
     '@media (max-width: 850px)': {
       fontSize: '13px',
     },
+
+    '*': {
+      margin: '0',
+      padding: '0',
+      border: '0',
+      // verticalAlign: 'baseline',
+    },

```

Comment:
Seems like this is what we want, is not?
```suggestion
      margin: '0',
      padding: '0',
      border: '0',
```